app/models/comment.rb

```diff
@@ -1,4 +1,11 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  after_create :update_comments_counter
+  after_destroy :update_comments_counter
+
+  def update_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
 end
```

Comment:
[Suggestion]: It would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_comments_counter method every time a new post has been saved to database, then make the update_comments_counter method private.